profile/README.md

```diff
@@ -1,31 +1,35 @@
 <div align="center">
-<a href="https://privacyguides.org#gh-light-mode-only">
-<img src="https://privacyguides.org/assets/img/layout/privacy-guides-logo.svg" width="500px" alt="Privacy Guides" />
-</a>
 
-<a href="https://privacyguides.org#gh-dark-mode-only">
-<img src="https://privacyguides.org/assets/img/layout/privacy-guides-logo-dark.svg" width="500px" alt="Privacy Guides" />
+<a href="https://www.privacyguides.org/">
+    <picture>
+        <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/100533118/169622905-58b29d79-afde-4e60-b35a-2e308246beee.svg">
+        <img alt="Privacy Guides" width="500px" src="https://user-images.githubusercontent.com/100533118/169622923-38edf4b3-1960-48ab-b440-63cc149f1bec.svg">
```

Comment:
Feels like we should keep:
```suggestion
        <source media="(prefers-color-scheme: dark)" srcset="https://privacyguides.org/assets/img/layout/privacy-guides-logo.svg">
        <img alt="Privacy Guides" width="500px" src="https://privacyguides.org/assets/img/layout/privacy-guides-logo.svg">
```

Reply:
This README is only for GitHub, so I think it would make sense to use the GitHub CDN as opposed to fetching them from PG. More optimized that way.